composer.lock

```diff
@@ -12562,5 +12562,5 @@
     "platform-overrides": {
         "php": "7.2.5"
     },
-    "plugin-api-version": "2.2.0"
+    "plugin-api-version": "2.3.0"
```

Comment:
Maybe we should keep 2.2.0?

Reply:
I don't know. Why using an old api version ?

Reply:
Because there's still a lot of projects that are not compatible with composer 2.3 and forcing them to use composer 2.3 make us switch between composer version

Reply:
@atomiix Reverted.

Reply:
Same issue again 😅

Reply:
@atomiix Reverted.

Reply:
I'm gonna stop fighting for this one as it's already on the 8.0.x branch 🥲

Reply:
Big sorry.